### DISCUSSION TITLE
Do SB and SD ads have similar limitations

### DISCUSSION THREAD

**Discussion by Lin1031** [amzn/ads-advanced-tools-docs#220] [Q&A]
Do SB and SD ads have similar limitations？If so, can you tell me

https://advertising.amazon.com/help/G86H2227323T8T4Q

**Comment by elizafarley** [comment]
Hi @Lin1031, we don't currently have those limits published in the documentation, but have opened an internal ticket in our backlog to add. Thanks for the request!

**Comment by Lin1031** [comment]
Hi @elizafarley , thank you for your response.

Prior to your inquiry, I reached out to customer service, and they informed me that the limitation of 10,000 ad campaigns is applicable to all types of advertisements. I would like to verify if this is indeed the case.

Furthermore, if there have been any updates regarding the restrictions on SD and SB in the documentation, could you please provide me with the updated information once again?

Thank you.

**Reply by Lin1031** [reply]
And I would like to know the following information as well:

What is the maximum number of keywords allowed in an account, including sponsored product, brand, or display campaigns?

Additionally, what are the maximum limits for category and product targets in an account, including sponsored product, brand, or display campaigns?

Thank you
